app/pages/Recovery/RecoveryIntroduction.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { push } from 'connected-react-router';
+import { useDispatch } from 'react-redux';
+import routes from '~/constants/routes.json';
+import Button from '~/cross-app-components/Button';
+
+import styles from './Recovery.module.scss';
+
+/**
+ * Displays information about the recovery.
+ */
+export default function RecoveryIntroduction() {
+    const dispatch = useDispatch();
+
+    return (
+        <div className={styles.introduction}>
+            <h2>Account Recovery</h2>
+            <p>
+                You can recover your lost accounts using a Ledger device that
```

Comment:
I think this text needs to revising as it is hard to understand. Also it says that data is stored on the Ledger, which is not really the case. I think we can merge without revising it, but I would like someone to help us make it better than it is now.